core/src/integration-test/java/com/scalar/db/storage/dynamo/DynamoWithReservedKeywordIntegrationTest.java

```diff
@@ -2,59 +2,33 @@
 
```

Comment:
Cause the type of concatenated clustering key changes from string to binary so those tests also need to be changed, so they are related to this PR otherwise the CI will be failed.